### DISCUSSION TITLE
CCDs to Epic Interconnect

### DISCUSSION THREAD

**Discussion by rkight66** [nextgenhealthcare/connect#5281] [General]
I'm trying to figure out how to generate the SOAP 1.2 wrapper around a CCD XML document to send to Epic Interconnect.  Has anyone attempted this before?
I'm working in Mirth Connect 3.8.1
An HTTP Sender, Method=POST
I'm lost after that.  Anyone done anything like this?

**Comment by siddharthsharma9258** [comment]
Normally, the receiver (Epic Interconnect in this case) provides the envelope. It could be embedded in a URL (WSDL), or sometimes in the form of schema files. Both options exist, you just need to see which one they have given you.

You don't need to use HTTP Sender, but Web service Sender in destination. Plug in the WSDL URL, provide your authentication, and select the operation/method you want to perform.

**Comment by joshmc82** [comment]
Agreed with the suggestion of getting the WSDL address. That would allow you to plug into a Web Service Sender and it would generate the envelope for you. Epic (or your customer) should be able to give you that WSDL if you are sending SOAP. You're likely going to need to produce an IHE ITI-41 Provide and Register message.
https://profiles.ihe.net/ITI/TF/Volume2/ITI-41.html
